### PR TITLE
Added support for eu-west-1 region

### PR DESCRIPTION
**Pull request by gileze33**

### CODE REVIEW

**Comment by gileze33** [comment]
Looks like the build failure is just due to circle missing your AWS details

File: packer.json [hunk 1]
```diff
@@ -2,6 +2,19 @@
   "builders": [
     {
       "type": "amazon-ebs",
+      "name": "empire-eu-west-1",
+      "access_key": "",
+      "secret_key": "",
+      "region": "eu-west-1",
+      "source_ami": "ami-47a23a30",
```
Comment (phobologic) [inline packer.json:9]: This is a fairly old AMI - may want to use a more modern one.  The most recent: ami-92401ce5

**Comment by phobologic** [comment]
Yeah - this is for security on CircleCI's part.  Otherwise someone could fork and push a build that printed out the credentials.  In order for me to get this kicked off, I'll need to create a branch based on your fork and push it.

Anyway, I'm going to wait to hear from you about updating the PR with a newer image, then I'll do what is necessary to get this tested.  Thanks for all this!

**Comment by phobologic** [comment]
Ping @gileze33

**Comment by gileze33** [comment]
Apologies, I've been busy migrating our production environment onto empire (exciting!). Will try to get this updated tomorrow morning.

**Comment by phobologic** [comment]
No worries!  That's awesome news!  Let us know if there's anything we can do to help, and how it goes.  Are you using the stacker stuff to build it out, or are you doing it another way?

**Pull request closed without merging**